### DISCUSSION TITLE
JSON comparison available when using the svelte-jsoneditor in my own project?

### DISCUSSION THREAD

**Discussion by SleepyPeel** [josdejong/svelte-jsoneditor#412] [Q&A] [answered]
If I would use the svelte-jsoneditor in my own project, would the JSON compare that is available in JSON Editor Online, also be included in my project?

**Comment by josdejong** [comment, marked as answer]
The JSON compare feature of JSON Editor Online is not a part of `svelte-jsonediter`.

**Reply by SleepyPeel** [reply]
Okay, thank you very much for the fast answer. :)
